templates/common/_base/units/machine-config-daemon-pull.service

```diff
@@ -12,11 +12,12 @@ contents: |
   Before=machine-config-daemon-host.service
 
   [Service]
-  # Need oneshot to delay kubelet
-  Type=oneshot
+  Type=simple
```

Comment:
> (there is no kubelet on base FCOS and it hasn't pivoted yet)

I don't quite get how this is related?  We need to block kubelet on RHCOS for sure.

Reply:
`fcos` branch, this is not going to master yet

Reply:
Right, I would just be curious to understand why you made this change.  It's not necessary for `Restart=` is it?